### PR TITLE
fix(dop): Solve the problem that the encrypted value is empty when batch update

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
Solve the problem that the encrypted value is empty when batch update

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?id=329181&iterationID=1337&pId=0&type=BUG)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     Solve the problem that the encrypted value is empty when batch update         |
| 🇨🇳 中文    |      解决批量修改的时候加密value为空问题        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/apps/dop/providers/cms/cicdcmsservice.go [hunk 1]
```diff
@@ -144,6 +118,41 @@ func (s *CICDCmsService) CICDCmsCreateOrUpdate(ctx context.Context, req *CICDCms
 	return true, nil
 }
 
+func GetKeysAndValueMap(req *CICDCmsCreateOrUpdateRequest) (keys []string, valueMap map[string]*cmspb.PipelineCmsConfigValue) {
```
Comment (sfwn) [inline internal/apps/dop/providers/cms/cicdcmsservice.go:121]: no need export this method.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/apps/dop/providers/cms/cicdcmsservice.go (hunk 1 above)
Comment (littlejiancc) [reply internal/apps/dop/providers/cms/cicdcmsservice.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**